lib/utils/is-file-type.js

```diff
@@ -0,0 +1,25 @@
+const imageSize = require('image-size');
+
+/**
+ * detect whether content is an image
+ * @param {Buffer} content
+ */
+function isImage(content) {
+  try {
+    imageSize(content);
+    return true;
+  } catch (err) {
+    return false;
+  }
+}
+
+/**
+ * detect whether content is an image
+ * @param {string} content
+ */
+function isSVG(content) {
+  return content.includes('<svg');
+}
+
+exports.isImage = isImage;
```

Comment:
This module file name (`is-file-type`) is misleading. I would expect that it exports single function (`isFileType`), but it actually the set of utilities to detect file type.

I guess that this file should be renamed to something more appropriate, or, this exports should be split as separate modules.